packages/kbn-unified-data-table/src/components/source_document.tsx

```diff
@@ -36,6 +36,7 @@ export function SourceDocument({
   isPlainRecord,
   fieldFormats,
   dataTestSubj = 'discoverCellDescriptionList',
+  style,
```

Comment:
Can we replace it with a custom class name or `css` prop? We don't use `style`.

Reply:
@jughosta Custom class mean we will have to keep the css somewhere which will not be a good idea as we are not managing any css at the moment.
css prop means passing string and then parsing it in the `source_document` component which i am not sure adds any value.
Do you foresee any challenge with the style prop ?

Reply:
Adding CSS styles directly to HTML elements is not considered to be a good practice. Emotion (via `css` prop) does not inline styles, it extracts them into generated class name definitions. I think both options (custom class name and `css` prop) would be better compatible with most browsers as they might run additional postprocessors for styles (depending on the configuration in Kibana).

Reply:
i keep on forgetting about the emotion way of passing css. Thank you for reminding